test/prime generator/prime.stdin

```diff
@@ -0,0 +1,6 @@
+2
+2
+10
+5
+50
```

Comment:
You might want to perform a run with an even larger set of primes to output and verify against, but that's up to you.